lib/core/base/audit.js

```diff
@@ -165,7 +165,7 @@ Audit.prototype.run = function (context, options, resolve, reject) {
 							result: axe.constants.CANTTELL,
 							description: 'An error occured while running this rule',
 							message: err.message,
-							help: err.stack || err.message,
```

Comment:
No. The reason I deleted this is because this actually didn't work. This `help` property is actually overwritten later, so this line did nothing.